### PR TITLE
Add warning about construction work in docs

### PR DESCRIPTION
**Pull request by BTCparadigm**
https://github.com/zkSNACKs/WasabiDoc/pull/1020#issuecomment-999512041

> Would not want to be a negative Nelly but looking at this repo now makes me even more confident that it's harder to change everything while not braking things too badly, than to use the other repo and copy required pieces to it one by one.

> Other thoughts:
> - Should we say Wasabi Wallet 2.0 everywhere? // No
> - Should we destroy for example "Why Wasabi" section to keep 2.0 docs simple? // All categories/sections need to remain as is to not break links. Only content should be updated.
> - Is the time period that docs would remain unreliable and confusing too much? // YOLO

### CODE REVIEW

**Comment by BTCparadigm** [comment]
Updated repository main README.md to 2.0 and to inform Github users that construction work has begun, so all info might not be up to date.
Also removed shields part because in 2.0 we have different shield.

**Comment by BTCparadigm** [comment]
Newest plans for docs in #1022 so forget my first comment.

This PR contains warnings for this repositorys README.md and for the websites [landing page](https://wasabiwallet.io/). 
It changes the website message from `config.js` module (I think), adds a picture and updates README.md's to 2.0.

These can't easily be added as dropdown box, hence screwing up with current in-use website is risky. I have not yet had time to test run this but PR is here in case someone wants to run it or add something.

This PR contains stupid commits because I was trying things out, like adding and deleting a pic.

File: README.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
 <p align="center">
   <a href="https://docs.wasabiwallet.io">
-    <img src="docs/.vuepress/public/WasabiWalletDocs.png" alt="Wasabi Wallet Documentation" title="Wasabi Wallet Documentation" class="Image" />
+    <img src="docs/.vuepress/public/Under_Construction.png" alt="Documentation under construction" title="Documentation under construction" class="Image" />
   </a>
 </p>
 
-[Wasabi Wallet](https://wasabiwallet.io) is an open-source, non-custodial, privacy focused Bitcoin wallet for desktop. It implements a trustless privacy technique, called [Chaumian CoinJoin](https://github.com/nopara73/ZeroLink/).
+## Note that [Wasabi Wallet documentation](https://docs.wasabiwallet.io/) is under renovation, which means some content you may want to work on is about to be obsoleted and some of the information you see in this documentation might not be up to date.
```
Comment (yahiheb) [inline README.md:7]:
```suggestion
### Note that [Wasabi Wallet documentation](https://docs.wasabiwallet.io/) is under renovation, some content is about to be obsoleted and some information might not be up to date.
```

File: README.md [hunk 2]
```diff
@@ -1,10 +1,12 @@
 <p align="center">
   <a href="https://docs.wasabiwallet.io">
-    <img src="docs/.vuepress/public/WasabiWalletDocs.png" alt="Wasabi Wallet Documentation" title="Wasabi Wallet Documentation" class="Image" />
```
Comment (yahiheb) [inline README.md:3]: When replacing a file or a picture, make sure it gets deleted if it is not used elsewhere.

File: README.md [hunk 3]
```diff
@@ -1,10 +1,12 @@
...
+    <img src="docs/.vuepress/public/Under_Construction.png" alt="Documentation under construction" title="Documentation under construction" class="Image" />
   </a>
 </p>
 
-[Wasabi Wallet](https://wasabiwallet.io) is an open-source, non-custodial, privacy focused Bitcoin wallet for desktop. It implements a trustless privacy technique, called [Chaumian CoinJoin](https://github.com/nopara73/ZeroLink/).
+## Note that [Wasabi Wallet documentation](https://docs.wasabiwallet.io/) is under renovation, which means some content you may want to work on is about to be obsoleted and some of the information you see in this documentation might not be up to date. 
+
+[Wasabi Wallet](https://wasabiwallet.io) is an open-source, non-custodial, privacy focused Bitcoin wallet for desktop. It implements a trustless privacy technique, called [WabiSabi CoinJoin](https://eprint.iacr.org/2021/206).
```
Comment (yahiheb) [inline README.md:9]: How about this?

```suggestion
[Wasabi Wallet](https://wasabiwallet.io) is an open-source, non-custodial, privacy focused Bitcoin wallet for desktop. It implements a trustless privacy technique, called [CoinJoin](https://en.bitcoin.it/Privacy#CoinJoin).
```

File: README.md [hunk 4]
```diff
@@ -1,10 +1,12 @@
 <p align="center">
   <a href="https://docs.wasabiwallet.io">
-    <img src="docs/.vuepress/public/WasabiWalletDocs.png" alt="Wasabi Wallet Documentation" title="Wasabi Wallet Documentation" class="Image" />
+    <img src="docs/.vuepress/public/Under_Construction.png" alt="Documentation under construction" title="Documentation under construction" class="Image" />
```
Comment (yahiheb) [inline README.md:3]: I don't think we should change the tags:

```suggestion
    <img src="docs/.vuepress/public/Under_Construction.png" alt="Wasabi Wallet Documentation" title="Wasabi Wallet Documentation" class="Image" />
```

File: docs/.vuepress/config.js [hunk 5]
```diff
@@ -16,8 +16,8 @@ const youtubeEmbed = (id, path) => `
 const themeColor = "#211b24"
 
 module.exports = {
-  title: "Wasabi Docs",
-  description: "An archive of knowledge about Wasabi Wallet, an open-source, non-custodial and privacy-focused Bitcoin wallet for desktop.",
+  title: "Docs under construction",
+  description: "Wasabi Wallet documentation is currently under construction because of the upcoming 2.0 release. Be aware that some of the content might not yet be up to date.",
```
Comment (yahiheb) [inline docs/.vuepress/config.js:20]: We shouldn't change these IMO. Maybe just add something that says it is under construction.

```suggestion
  title: "Wasabi Docs",
  description: "An archive of knowledge about Wasabi Wallet, an open-source, non-custodial and privacy-focused Bitcoin wallet for desktop.",
```

File: docs/README.md [hunk 6]
```diff
@@ -1,16 +1,16 @@
 ---
 home: true
-heroText: Wasabi Documentation
-heroImage: /Logo_without_text.png
-actionText: Reclaim Your Privacy →
-actionLink: /using-wasabi/ELI5.md
+heroText: Wasabi Documentation is under construction
+heroImage: /Under_Construction.png
+actionText: Documentation under construction →
+actionLink: /building-wasabi/README.md
```
Comment (yahiheb) [inline docs/README.md:6]: We shouldn't change this IMO.

```suggestion
heroText: Wasabi Documentation
heroImage: /Logo_without_text.png
actionText: Reclaim Your Privacy →
actionLink: /using-wasabi/ELI5.md
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

**Review by yahiheb — APPROVED** [review]

**Comment by BTCparadigm** [comment]
Change in plans. Docs 2.0 will be built in [another branch](https://github.com/zkSNACKs/WasabiDoc/tree/docs-2.0).
CLosing this.
